app/routes/technical-help.js

```diff
@@ -12,65 +12,66 @@ module.exports = function (router) {
 
   router.post('/help', function (req, res, next) {
     try {
-      if (config.ZENDESK_ENABLED === 'true') {
-        const technicalHelp = new TechnicalHelp(
-          req.body.name,
-          req.body.emailAddress,
-          req.body.referenceNumber,
-          req.body['date-of-claim-day'],
-          req.body['date-of-claim-month'],
-          req.body['date-of-claim-year'],
-          req.body.issue
-        )
+      const technicalHelp = new TechnicalHelp(
```

Comment:
I've shuffled this around a little so that if `ZENDESK_ENABLED` is `false` (e.g. on local dev) it still tries to do form validation, build up the data for the ticket and log details. (Details only logged for debugging if the ticket **isn't** going to be created.)